File: natas11.py
import base64
import json

defdata = {"showpassword":"no","bgcolor":"#ffffff"}
#defdata = {"showpassword":"yes","bgcolor":"#ffffff"}
keydata = base64.b64decode("ClVLIh4ASCsCBE8lAxMacFMZV2hdVVotEhhUJQNVAmhSEV4sFxFeaAw=").decode("UTF-8")
#keydata = "qw8J"
textdata = json.dumps(defdata)
outtext = ""

for i in range(len(textdata)):
    ti = textdata[i]
    ki = keydata[i%len(keydata)]
    outtext += chr(ord(ti)^ord(ki))
print(outtext)
#print(base64.b64encode(outtext.encode('ascii')))

File: natas11.php
<?php

$defaultdata = array( "showpassword"=>"no", "bgcolor"=>"#ffffff");
$key = base64_decode("ClVLIh4ASCsCBE8lAxMacFMZV2hdVVotEhhUJQNVAmhSEV4sFxFeaAw=");
$text = json_encode($defaultdata);
$outText = "";

for($i=0;$i<strlen($text);$i++) {
    $outText .= $text[$i] ^ $key[$i % strlen($key)];
}
echo  $outText ;

?>